src/main/Port.Adapter/UI/Views/Blazor.Common/TreeView.razor

```diff
@@ -37,9 +38,9 @@
                                         <Blazorise.Icon Name="FontAwesomeIcons.InfoCircle" />
                                     </button>
                                     @if (
-                                        !string.IsNullOrWhiteSpace(child.Neuron.ExternalReferenceUrl) ||
-                                        (child.Neuron.Terminal != null && !string.IsNullOrWhiteSpace(child.Neuron.Terminal.ExternalReferenceUrl))
-                                    )
+                        !string.IsNullOrWhiteSpace(child.Neuron.ExternalReferenceUrl) ||
+                        (child.Neuron.Terminal != null && !string.IsNullOrWhiteSpace(child.Neuron.Terminal.ExternalReferenceUrl))
+                        )
                                     {
```

Comment:
Can the tabs be restored so it is aligned with the rest of the code?

Reply:
Done

Reply:
Please add a space after the || operator.

Reply:
Done